core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -279,6 +285,7 @@ void removeBuddyInfoForRemovedUser(User user, Buddy buddy)
 		{
 			// Buddy request was accepted
 			removeMessagesSentByBuddy(user, buddy);
+			removeMessagesSentByUserToBuddy(user, buddy);
```

Comment:
Yes, that's the case. The failing test was ``OverwriteUserTest.'Overwrite Richard while being a buddy of Bob and verify that Bob can move on with everything'``. The other one that failed is ``OverwriteUserTest.'Overwrite Richard while being a buddy of Bob and verify Richard does not receive Bob\'s goal conflicts anymore'``